stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -679,13 +711,20 @@ class Stripe internal constructor(
      * See [Create a card token](https://stripe.com/docs/api/tokens/create_card).
      *
      * @param card the [Card] used to create this payment token
+     * @param idempotencyKey optional, see [Idempotent Requests](https://stripe.com/docs/api/idempotent_requests)
      * @param callback a [ApiResultCallback] to receive the result or error
      */
     @UiThread
-    fun createToken(card: Card, callback: ApiResultCallback<Token>) {
+    @JvmOverloads
+    fun createToken(
```

Comment:
Out of curiosity... how does @JvmOverloads work for a function like this, where the 2nd arg is optional -- but the 3rd is not? Does a Java caller always need to specify `idempotencyKey` since it is positionally before a non optional argument?

Reply:
@markfi-stripe good question. I verified that it works as intended - `idempotencyKey` becomes optional.

<img width="885" alt="Screen Shot 2019-11-03 at 6 11 16 PM" src="https://user-images.githubusercontent.com/45020849/68093598-615abf80-fe65-11e9-89d8-2916b293d00b.png">